low-level/files/opt/seagate/sspl/bin/consul_config.json

```diff
@@ -1,9 +1,9 @@
 {
-    "watches": [
-          {
-            "type": "key",
-            "key": "sspl/config/SYSTEM_INFORMATION/log_level",
-            "args": ["/opt/seagate/cortx/sspl/bin/consume_cfg_change_alert"]
-          }
-    ]
+  "watches": [
+        {
+          "type": "key",
+          "key": "sspl/config/SYSTEM_INFORMATION/log_level",
+          "args": ["/opt/seagate/cortx/sspl/low-level/files/opt/seagate/sspl/bin/consume_cfg_change_alert"]
```

Comment:
again, deployment path should not change
/opt/seagate/cortx/sspl/bin/ should remain same